include/ur_client_library/rtde/rtde_client.h

```diff
@@ -74,7 +74,8 @@ enum class UrRtdeSafetyStatusBits
   IS_EMERGENCY_STOPPED = 7,
   IS_VIOLATION = 8,
   IS_FAULT = 9,
-  IS_STOPPED_DUE_TO_SAFETY = 10
+  IS_STOPPED_DUE_TO_SAFETY = 10,
+  IS_SYSTEM_THREE_POSITION_ENABLING_STOPPED = 11
```

Comment:
I would suggest following the RTDE documentation regarding the naming:
```suggestion
  IS_3PE_INPUT_ACTIVE = 11
```

Reply:
Thanks @urfeex for the feedback.

So, I picked this name by analogy with what I found in `SafetyStatus`: `SafetyStatus::ROBOT_EMERGENCY_STOP` goes to `UrRtdeSafetyStatusBits::IS_ROBOT_EMERGENCY_STOPPED`, so I analogized `SafetyStatus::SYSTEM_THREE_POSITION_ENABLING_STOP` to the proposed `UrRtdeSafetyStatusBits::IS_SYSTEM_THREE_POSITION_ENABLING_STOPPED`. Just let me know which you prefer. If you still prefer the one you proposed here over the analogized one, I'll just accept your edit and you can merge.

Reply:
Yes, I figured that's how you picked the name. However, they don't refer to the same thing exactly:

- There is also TP_THREE_POSITION_ENABLING_STOP. I don't know what the dashboard reports as SYSTEM_THREE_POSITION_ENABLING_STOP, but the RTDE field we're talking about is a boolean AND of the teach pendant 3pe and an external 3pe depending on which is present.
- I also don't know whether the STOPPED one from the dashboard server is actually the inverse of the INPUT_ACTIVE

We always try to be as close to the public documentation as possible to avoid uncertainties.

Reply:
@urfeex - OK, I applied your suggested edit.